src/ipc/StrandCoord.h

```diff
@@ -32,16 +33,20 @@ class StrandCoord
     String tag; ///< optional unique well-known key (e.g., cache_dir path)
 };
 
-/// strand registration with Coordinator (also used as an ACK)
-class HereIamMessage
+/// an IPC message carrying just the kid coordinates and the message kind
+class StrandMessage
 {
 public:
-    explicit HereIamMessage(const StrandCoord &strand); ///< from registrant
-    explicit HereIamMessage(const TypedMsgHdr &hdrMsg); ///< from recvmsg()
-    void pack(TypedMsgHdr &hdrMsg) const; ///< prepare for sendmsg()
+    StrandMessage(MessageType, const StrandCoord &);
+    explicit StrandMessage(const TypedMsgHdr &);
+    void pack(TypedMsgHdr &) const;
+
+    /// creates and sends StrandMessage to Coordinator
+    static void NotifyCoordinator(const MessageType, const char *tag);
```

Comment:
We use `const` in implementation. Some compilers warn about "unnecessary" `const` in declarations.

```suggestion
    static void NotifyCoordinator(MessageType, const char *tag);
```